package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "token-metadata-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "rimraf ./dist && tsc --project tsconfig.build.json",
+    "start": "node dist/index.js",
+    "start-ts": "ts-node ./src/index.ts",
+    "test": "jest",
+    "testenv:run": "docker-compose -f docker/docker-compose.dev.postgres.yml up",
+    "testenv:stop": "docker-compose -f docker/docker-compose.dev.postgres.yml down -v -t 0",
+    "testenv:logs": "docker-compose -f docker/docker-compose.dev.postgres.yml logs -t -f",
+    "migrate": "ts-node node_modules/.bin/node-pg-migrate -j ts",
+    "lint:eslint": "eslint . --ext .js,.jsx,.ts,.tsx -f unix",
+    "lint:prettier": "prettier --check src/**/*.ts tests/**/*.ts migrations/**/*.ts",
+    "generate:openapi": "node -r ts-node/register ./util/openapi-generator.ts",
+    "generate:docs": "redoc-cli build --output ./tmp/index.html ./tmp/openapi.yaml"
+  },
+  "author": "",
+  "license": "ISC",
+  "prettier": "@stacks/prettier-config",
+  "devDependencies": {
+    "@babel/core": "^7.20.2",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-typescript": "^7.18.6",
+    "@stacks/eslint-config": "^1.2.0",
+    "@types/jest": "^28.1.8",
+    "@typescript-eslint/eslint-plugin": "^5.43.0",
+    "@typescript-eslint/parser": "^5.43.0",
+    "babel-jest": "^28.1.3",
+    "eslint": "^8.27.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-tsdoc": "^0.2.17",
+    "jest": "^28.1.3",
+    "prettier": "^2.7.1",
+    "redoc-cli": "^0.13.20",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^28.0.8",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
+  },
+  "dependencies": {
+    "@fastify/cors": "^8.2.0",
+    "@fastify/swagger": "^7.6.1",
+    "@fastify/type-provider-typebox": "^2.3.0",
+    "@sinclair/typebox": "^0.24.51",
+    "@stacks/transactions": "^4.3.8",
```

Comment:
In the API, we had a few hot-paths that were decoding clarity values which ended up becoming CPU bottlenecks. If you think this service might run into the same problem, you can use https://github.com/hirosystems/stacks-encoding-native-js which was created to solve the issue.